karma.conf.js

```diff
@@ -48,7 +48,7 @@ module.exports = function(config) {
       'app/bower_components/angularfire/dist/angularfire.js',
       'app/bower_components/restangular/dist/restangular.js',
       'app/bower_components/moment/moment.js',
-      'app/bower_components/moment-timezone/builds/moment-timezone-with-data.js',
+      'app/bower_components/moment-timezone/builds/moment-timezone-with-data-2012-2022.js',
```

Comment:
They've had it this way for some time and are looking to remove the date in a future version.

I had the specific data file in there before, switched it to the full one (which is considerably larger) so tests would work, now switching it back to what it was before.